### ISSUE TITLE
Feature request: Support images inside page comments

### ISSUE THREAD

**Issue by petarlatinovic** [BookStackApp/BookStack#5327]
### Describe the feature you'd like

We currently use bookstack to provide technical documentation to our customers. We also have a number of special projects that our community supports and provides feedback. This feedback used to be provided using Easy Forums plugin before we retired Confluence with Bookstack. As an alernative, to get feedback, we use the comments functionality inside Bookstack which meets 99% of our use case. The problem we have is that the communication is somewhat limited as the comments \ questions posed cannot include the pictures of the specific area the feedback is provided. This is also limited in our replies. As an alternative, our customers are emailing us screenshots to supplement the feedback, which breaks the loop and conversations are sometimes lost.

### Describe the benefits this would bring to existing BookStack users

- Communication is contained within bookstack, and supplementary evidence does not need to be provided via email
- For our use case, it removes the need to have a forum type product
- Encourages the use of bookstack app within our organsiation \ one stop shop

### Can the goal of this request already be achieved via other means?

No, not without deploying circumventing Bookstack.

### Have you searched for an existing open/closed issue?

- [X] I have searched for existing issues and none cover my fundamental request

### How long have you been using BookStack?

1 to 5 years

### Additional context

We all appreciate what you're doing and how you're supporting the overall community.

**Comment by s375502** [comment]
Yep, surprised this function does not exist, would be great to throw screenshots etc into replies

**Comment by jferriggi** [comment]
This would be a really cool feature.  It would make Bookstack even better!

**Comment by My-Random-Thoughts** [comment]
I agree it's a great feature but would need to be configurable.

There could be instances where someone uploads massive 4k images into a comment or two and forces a server to be out of space.  Limits on sizes could be implemented.

**Comment by jkarolczak** [comment]
Upvoting this - hope it gets added!

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/5327
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
